pom.xml

```diff
@@ -391,6 +390,20 @@
 					</plugin>
 				</plugins>
 			</build>
+			<properties>
+				<p2.baselineMode>failCommon</p2.baselineMode>
+			</properties>
+		</profile>
+		<profile>
+			<id>baseline-comparison-strict</id>
+			<activation>
+				<property>
+					<name>CI</name>
```

Comment:
Why only on CI?

Reply:
> Because when one runs the build on Windows all text-based source files are different due to different line-endings.

OK. Can't we find some option to force the line-ending (and remove timestamps) in the code generator so this is portable?
If the issue is the OS only then instead of a property the activation should be based on the os.

Reply:
> > Because when one runs the build on Windows all text-based source files are different due to different line-endings.
> 
> OK. Can't we find some option to force the line-ending (and remove timestamps) in the code generator so this is portable? 

I have not found an option to disable that in modello. But I since only the java source files are ignored and not the class files I think it is OK to ignore differences in the sources. If there are changes that lead to a binary difference they will be noticed in the class-files.
Regarding the text files, do you know a elegant way to achieve that? I only found some suggestions to use the resources plug-in and define different resource sets. But that seems a bit inconvenient. On the other hand just accepting that and only warn about differences like it is at the moment by default is also ok.


> If the issue is the OS only then instead of a property the activation should be based on the os.

I would prefer enable strict comparison explicitly, so that it does not suddenly disabled (without anybody noticing).
Enabling that with the `eclipse-sign` looks like a suitable place to me. This is only enabled on Jenkins which also creates the released bits. The chances of false positive differences are the lowest here.